dpypelines/pipeline/shared/transforms/sanity_check.py

```diff
@@ -0,0 +1,16 @@
+from pathlib import Path
+from typing import List
+
+import pandas as pd
+
+
+def sdmx_sanity_check_v1(sdmx_file: Path):
+    """
+    Sanity check that the recieved sdmx file is actually an sdmx file.
+    """
+    assert sdmx_file.name.endswith(".xml"), "Invalid sdmx_file"
+
+    try:
+        pd.read_xml(sdmx_file, nrows=10)
```

Comment:
The function fails for me. From my end 'nrows' is not a valid kwarg for read_xml(), does it work for you or is it a python version issue?